### ISSUE TITLE
UTF-8 problem

### ISSUE THREAD

**Issue by acepow3r** [istvan-v/plus4emu#1]
I discovered the same problem that plagued ep128emu is present at plus4emu, too.
Whenever I try to insert a disk image located in a folder or path with a (greek) localized name eg. D:\τεστ\ROMs gives an error.

**Comment by istvan-v** [comment]
Yes, the same UTF-8 fixes are already in the Git source code, there are just no binary packages yet that include them. However, I plan to release an updated version soon.
